pkgs/development/compilers/llvm/common/compiler-rt/default.nix

```diff
@@ -82,15 +83,24 @@ stdenv.mkDerivation ({
     "-DCMAKE_ASM_COMPILER_TARGET=${stdenv.hostPlatform.config}"
   ] ++ lib.optionals (haveLibc && stdenv.hostPlatform.libc == "glibc") [
     "-DSANITIZER_COMMON_CFLAGS=-I${libxcrypt}/include"
-  ] ++ lib.optionals ((useLLVM || bareMetal || isMusl || isAarch64) && (lib.versions.major release_version == "13")) [
+  ] ++ lib.optionals (useLLVM && haveLibc && stdenv.cc.libcxx == libcxx) [
+    "-DSANITIZER_CXX_ABI=libcxxabi"
```

Comment:
Please use the CMake functions in lib instead of writing out the full the.

Reply:
Note then I need to set the type in the variable, `:something` no?

I sort of think it might be best to clean up this flag list an a second PR to staging. My goal here was mainly to avoid a mass rebuild.

Reply:
Gotcha, I was thinking of making a PR that would switch over to the functions in lib so we can probably do that all in one go.

Reply:
Great, thank you!